diff --git a/pyproject.toml b/pyproject.toml
index 88177d13..02f15f13 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -10,24 +10,24 @@ authors = [
     {name = "Luca Lista", email = "luka.lista@gmail.com"}
 ]
 readme = "README.md"
-requires-python = ">=3.12"
-license = "LGPL-3.0"
+requires-python = ">=3.10"
+license = {text = "LGPL-3.0"}
 classifiers = [
     "Programming Language :: Python :: 3",
     "Operating System :: OS Independent",
 ]
 dependencies = [
     "imagecodecs",
-    "ipywidgets",
+    # "ipywidgets",  # skipped for scAnt
     "jsonpickle",
     "matplotlib",
-    "numpy>=2.4.1",
+    "numpy",
     "opencv-python-headless>=4.11",
     "pillow>=12.2.0",
     "psdtags",
     "psutil",
     "Pygments>=2.20.0",
-    "PySide6",
+    # "PySide6",  # skipped for scAnt
     "pytest>=9.0.3",
     "rawpy",
     "scipy",
